### DISCUSSION TITLE
How to get master coordinates from variable font

### DISCUSSION THREAD

**Discussion by ollimeier** [fonttools/fonttools#2546] [Q&A] [answered]
Hi,

I am wondering if there is a function to get the coordinates of the used masters from a variable font.

It's easy for predefined instances to get the coordinates from the fvar table, but sometimes the master values differ slightly from the specified instances. Is it somehow possible to I get the masters from a variable font? Any ideas? 

I am wondering if it's possible to get the masters when looking into the gvar table? 

Thanks in advance, Olli

**Comment by justvanrossum** [comment]
Yes, the master location coordinates for glyph shapes can be found (in normalized form) in the gvar table. they can be different per glyph. Advance widths/heights (HVAR/VVAR) can also have different master location coordinates. GPOS data also (but it's stored in GDEF). MVAR ditto.

**Comment by anthrotype** [comment, marked as answer]
I hacked something for getting gvar master locations here: https://gist.github.com/anthrotype/a1498f5eefd747ffa47ae4d421e905a7
